tests/flow/arith/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -162,7 +162,7 @@ let tests = [
     x + \\"\\"; // error
     \\"\\" + x; // error
     x + {}; // error
-    ({}) + x; // error
+    (({}) + x); // error
```

Comment:
this seems to be over eager for parenthesis here, but I don't think that it's a big deal. We'll likely never have an expression statement that starts with {} in practice.